fastlane/Fastfile

```diff
@@ -549,17 +548,46 @@ ENV["APP_STORE_STRINGS_FILE_NAME"]="AppStoreStrings.pot"
       download_path: "./fastlane/metadata"
     )
 
-    # Create the marketing and privacy URL files
-    metadata_locales.each { |locale|
-      marketing_path = Dir.pwd + "/metadata/#{locale}/marketing_url.txt"
-      write_file(marketing_path, 'https://woocommerce.com/mobile/')
-      
-      privacy_path = Dir.pwd + "/metadata/#{locale}/privacy_url.txt"
-      write_file(privacy_path, "https://automattic.com/privacy/")
+    # Ensure that localizations are present in the right places (some locale codes aren't correct)
+    {
+      "ar" => "ar-SA",
+      "de" => "de-DE",
+      "es" => "es-ES",
+      "fr" => "fr-FR",
+      "nl" => "nl-NL",
+      "zh-cn" => "zh-Hans",
+      "zh-tw" => "zh-Hant"
+    }.each { |source, destination|
+      source_dir = Dir.pwd + "/metadata/" + source
+      Dir.children(source_dir).select { |file| 
+        FileUtils.mv source_dir + "/" + file, Dir.pwd + "/metadata/" + destination + "/" + file
+      }
+      FileUtils.remove_dir(source_dir)
+    }
+
+    # Copy source files into `en-US`
+    enUS_path = Dir.pwd + "/metadata/en-US/"
+
+    FileUtils.mkdir_p(enUS_path) unless File.directory?(enUS_path)
```

Comment:
I'm not entirely sure the `unless it exists` test is necessary with `mkdir_p` – I don't think the `-p` option fails if destination exists already?

Reply:
You're right – turns out it's not necessary. Fixed in 49ee666b68863303a8b29c33f6a4aba413e27109